test/parallel/test-spawn-cmd-named-pipe.js

```diff
@@ -42,10 +42,8 @@ if (!process.argv[2]) {
     assert.fail('Failed to get COMSPEC');
   }
 
-  const args = ['/c', process.execPath, __filename, 'child',
-                '<', stdinPipeName, '>', stdoutPipeName];
-
-  const child = spawn(comspec, args);
+  const child = exec(`"${process.execPath}" "${__filename}" child < ${
```

Comment:
This does not work:
```js
  const args = ['/c', `"${process.execPath}"`, `"${__filename}"`, 'child',
                '<', stdinPipeName, '>', stdoutPipeName];

  const child = spawn(comspec, args,
                      { shell: true, windowsVerbatimArguments: true });
```
Should I prefer on of the beforementioned variants with `spawn()`?